### PR TITLE
Use checkbox for 'Include in Backup' column

### PR DESCRIPTION
**Pull request by donob4n**
It was really simple thanks to model/view architecture.

I tried to center the checkbox on the column but it will require to subclass ItemDelegate or similar. Maybe it would be just fine with a short column header text (e.g. "Inc. in Backup") or using two lines instead one.

### CODE REVIEW

**Comment by marmarek** [comment]
> "Inc. in Backup"

Maybe just "Backup"? Possibly with a tooltip "Include in Backup"?

File: qubesmanager/qube_manager.py [hunk 1]
```diff
@@ -477,6 +472,27 @@ def headerData(self, col, orientation, role):
...
 
+    def setData(self, index, value, role=Qt.EditRole):
+        if not index.isValid():
+            return False
+
+        if role == Qt.CheckStateRole:
+            col_name = self.columns_indices[index.column()]
+            if col_name == "Include in backups":
+                vm = self.qubes_cache.get_vm(index.row())
+                vm.vm.include_in_backups = (value == 2)
```
Comment (marmarek) [inline qubesmanager/qube_manager.py:483]: What is value `2`? Isn't there some nice constant like `Qt.Checked`?

**Review by marmarek — CHANGES_REQUESTED** [review]

**Comment by donob4n** [comment]
> > "Inc. in Backup"
> 
> Maybe just "Backup"? Possibly with a tooltip "Include in Backup"?

With just 'Backup' looks nice. I'm not sure about what to do with the tooltip. No other headers or cells have tooltips so it can be hard to found. In case you prefer to add it, where do you see more useful in the column header or in all 'Backup' cells?

**Comment by marmarek** [comment]
PipelineRetry

**Comment by marmarek** [comment]
PipelineRetry

**Review by marmarek — APPROVED** [review]

**Pull request merged**